recipes/whisper-cpp/all/test_package/test_package.cpp

```diff
@@ -0,0 +1,17 @@
+#include <iostream>
+#include <memory>
+#include <vector>
+#include "whisper.h"
+
+
+int main() {
+    auto context = std::unique_ptr<whisper_context, void(*)(whisper_context*)>(
+            whisper_init_from_file("models/for-tests-ggml-base.en.bin"), &whisper_free);
+
+    if (context == nullptr) {
```

Comment:
Does this file come from the downloaded tar?

Reply:
Yes this file does